src/Histogram/Histogram.js

```diff
@@ -275,23 +275,20 @@ export class Histogram extends PureComponent {
         const brushedDomainMin = dateToTimestamp(brushedDomain[0]);
         const brushedDomainMax = dateToTimestamp(brushedDomain[1]);
 
-        if (brushedDomainMin !== this.state.brushTimeDomain.min
```

Comment:
Removing this check will make the histogram slower and scale worse for a large number of points, why do we need to we need to do it?